### PR TITLE
Remove dead code trying to terminate executions

### PR DESCRIPTION
**Pull request by narape**
# Hey, I just made a Pull Request!

## Description
Remove dead code trying to terminate executions on ERROR state.

## Motivation and Context
Once RuneState reach ERROR it won't be active neither passed through the handlers, so any code in the Handler 
for reacting to ERROR is futile and wont be ever run.

## Have you tested this? If so, how?
I have included unit tests.

## Checklist for PR author(s)

- [ ] Changes are covered by unit test
- [ ] All tests pass
- [ ] Code coverage check passes
- [ ] Error handling is tested
- [ ] Errors are handled at the appropriate layer
- [ ] Errors that cannot be handled where they occur are propagated
- [ ] (optional) Changes are covered by system test
- [ ] Relevant documentation updated
- [ ] This PR has NO breaking change to public API
- [ ] This PR has breaking change to public API and it is documented

## Checklist for PR reviewer(s)

- [ ] This PR has been incorporated in release note for the coming version
- [ ] Risky changes introduced by this PR have been all considered

### CODE REVIEW

**Comment by narape** [comment]
Depends on  #860

**Review by honnix — COMMENTED** [review]
Comments are mostly around unit test.

File: styx-flyte-client/src/test/java/com/spotify/styx/flyte/client/TestAdminService.java [hunk 1]
```diff
@@ -76,6 +77,7 @@ public void terminateExecution(final ExecutionOuterClass.ExecutionTerminateReque
   @Override
   public void listExecutions(final Common.ResourceListRequest request,
                              final StreamObserver<ExecutionOuterClass.ExecutionList> responseObserver) {
+    lastExecutionRequest = request;
```
[the remark this answers is not in the document]
Comment (narape) [reply styx-flyte-client/src/test/java/com/spotify/styx/flyte/client/TestAdminService.java]: Yeah good idea, test will become much better this way

**Comment by narape** [comment]
I will close this pr to open it again. it should show that just one commit on top of master

**Pull request closed without merging**